File: CodeForces/contest-2137/2137_A.fs
open System

let solve k x = x <<< (k-1)

let readInts () =
    let mutable line = Console.ReadLine()
    while (line <> null && line.Trim().Length = 0) do
        line <- Console.ReadLine()
    if isNull line then [||]
    else line.Split([| ' '; '\t' |], StringSplitOptions.RemoveEmptyEntries) |> Array.map int

[<EntryPoint>]
let main _argv =
    printfn "Hello, F#!"

    let t = Console.ReadLine() |> int
    for _ in 1 .. t do
        let arr = readInts()
        let k = arr.[0]
        let x = arr.[1]
        solve k x |> printfn "%d"
    0

File: CodeForces/contest-2137/2137_A.cs
using System;
using System.Text;

class Program
{
    static int Solve(int k, int x) => x << (k-2);

    static void Main()
    {
        string? line = Console.ReadLine();
        if (line == null)
            return;
        int t = int.Parse(line);
        var sb = new StringBuilder();
        while (t-- > 0)
        {
            string? ln = Console.ReadLine();
            while (ln != null && ln.Trim().Length == 0)
                ln = Console.ReadLine();
            if (ln == null)
                break;
            var parts = ln.Split((char[])null!, StringSplitOptions.RemoveEmptyEntries);
            int k = int.Parse(parts[0]);
            int x = int.Parse(parts[1]);
            sb.AppendLine(Solve(k, x).ToString());
        }
        Console.Write(sb.ToString());
    }
}
